### PR TITLE
Check unsupported abis.

### PR DESCRIPTION
**Pull request by ankur2136**
Sample 1
Screenshot 2024-05-06 at 11 17 58 AM

Sample 2
Screenshot 2024-05-06 at 11 18 10 AM

### CODE REVIEW

File: packages/cli/src/config/PublishDetails.ts [hunk 1]
```diff
@@ -343,6 +345,35 @@ const getAndroidDetails = async (
...
+    const x8664libs = [...stdout.matchAll(/lib\/x86_64\/(.*)/g)].flatMap(permission => permission[1]);
+    if (amV7libs.length > 0 || x86libs.length > 0 || x8664libs.length > 0) {
+
+      const messages = [
+        `Solana dApp Store only supports arm64-v8a abi.`,
+        `Your apk file contains following unsupported abis`,
+        ... amV7libs.length > 0 ? [`\narmeabi-v7a:\n` + amV7libs] : [],
+        ... x86libs.length > 0 ? [`\nx86:\n` + x86libs] : [],
+        ... x8664libs.length > 0 ? [`\nx86_64:\n` + x8664libs] : [],
+        `\n\nAlthough your app might work, these library files are unused and increase the size of apk file making the user experience worse while installing/updating your app.`,
```
Comment (sdlaver) [inline packages/cli/src/config/PublishDetails.ts:362]: "might" will likely worry developers who don't necessarily understand ABIs. I'd rephrase this to emphasize that this isn't a functionality issue, but rather a usability issue causing longer download times for users.

**Review by sdlaver — CHANGES_REQUESTED** [review]

File: packages/cli/src/config/PublishDetails.ts (hunk 1 above)
Comment (ankur2136) [reply packages/cli/src/config/PublishDetails.ts]: Updated.

**Review by sdlaver — APPROVED** [review]
